src/ModuleTestLite.cs

```diff
@@ -535,6 +535,18 @@ public override void OnAwake()
 			Initialise();
 		}
 
+		public override void OnStartFinished(StartState state)
+		{
+			base.OnStart(state);
+			if (HighLogic.LoadedSceneIsFlight)
+				GameEvents.OnGameSettingsApplied.Add(Initialise);
+		}
+
+		public void OnDestroy()
+		{
+			GameEvents.OnGameSettingsApplied.Remove(Initialise);
```

Comment:
Just to check — this won't produce scary error messages if it was never added, right?

Reply:
Nope, it won't say anything if there's no listener to remove.